build.sbt

```diff
@@ -986,9 +986,9 @@ lazy val `maven-launcher` = (project in file("dev") / "maven-launcher")
     )
 
 def scriptedSettings: Seq[Setting[_]] = ScriptedPlugin.scriptedSettings ++ 
-  Seq(scriptedLaunchOpts += s"-Dproject.version=$version.value") ++
+  Seq(scriptedLaunchOpts += s"-Dproject.version=${version.value}") ++
   Seq(
```

Comment:
This rule rewrite introduced an error that only appeared when `scripted` tsts where recovered (see change in following line).